covfee/client/admin/hit_block/hit_block.tsx

```diff
@@ -177,10 +183,11 @@ export const HitBlock = (props: Props) => {
 
               <ul>
                 {props.hit.journeys.map((journey, index) => {
-
                   let journeyNodes: NodeType[] = []
-                  for (const node_idx of journey.nodes) {
-                    journeyNodes.push(props.hit.nodes[node_idx - 1])
+                  for (const node_id of journey.nodes) {
+                    journeyNodes.push(
+                      props.hit.nodes.find((node) => node.id === node_id)
```

Comment:
Wasn't there a journey associations attribute or something like that, so that we wouldn't have to do the `find`?